Backend/app/rider-platform/rider-app/Main/src/Domain/Types/Booking/API.hs

```diff
@@ -425,14 +438,14 @@ buildBookingAPIEntity booking personId = do
 --Note :- if you are adding and extra field in BookingStatusAPIEntity then add it in BookingAPIEntity as well
 buildBookingStatusAPIEntity :: (CacheFlow m r, EsqDBFlow m r, EsqDBReplicaFlow m r) => Booking -> m BookingStatusAPIEntity
 buildBookingStatusAPIEntity booking = do
-  mbActiveRide <- runInReplica $ QRide.findActiveByRBId booking.id
+  mbActiveRide <- runInReplica $ QRideLite.findActiveByRBIdLite booking.id
   let showPrevDropLocationLatLon = maybe False (.showDriversPreviousRideDropLoc) mbActiveRide
       driversPreviousRideDropLocLat = if showPrevDropLocationLatLon then fmap (.lat) (mbActiveRide >>= (.driversPreviousRideDropLoc)) else Nothing
       driversPreviousRideDropLocLon = if showPrevDropLocationLatLon then fmap (.lon) (mbActiveRide >>= (.driversPreviousRideDropLoc)) else Nothing
       rideStatus = fmap (.status) mbActiveRide
       estimatedEndTimeRange = mbActiveRide >>= (.estimatedEndTimeRange)
       driverArrivalTime = mbActiveRide >>= (.driverArrivalTime)
-  sosStatus <- getActiveSos mbActiveRide booking.riderId
```

Comment:
What about older function?

Reply:
it's being used somePlace that's expecting Original ride to created this new one with new domain